Containerfile

```diff
@@ -54,73 +54,91 @@ RUN cargo build \
     --bin shuttle-next -F next
 
 
-# Base image for running each "shuttle-..." binary
-ARG RUSTUP_TOOLCHAIN
-FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-crate-base
-ARG folder
-# Some crates need additional libs
-COPY ${folder}/*.so /usr/lib/
-ENV LD_LIBRARY_PATH=/usr/lib/
-ENTRYPOINT ["/usr/local/bin/service"]
-
+####### Targets for each crate
 
-# Targets for each crate
-# Copying of each binary is non-DRY to allow other steps to be cached
-
-FROM shuttle-crate-base AS shuttle-auth
+#### AUTH
+FROM docker.io/library/debian:bookworm-20230904-slim AS shuttle-auth
 ARG CARGO_PROFILE
-COPY --from=chef-builder /build/target/${CARGO_PROFILE}/shuttle-auth /usr/local/bin/service
+COPY --from=chef-builder /build/target/${CARGO_PROFILE}/shuttle-auth /usr/local/bin
+ENTRYPOINT ["/usr/local/bin/shuttle-auth"]
 FROM shuttle-auth AS shuttle-auth-dev
 
-FROM shuttle-crate-base AS shuttle-builder
+
+#### BUILDER
+ARG RUSTUP_TOOLCHAIN
+FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-bookworm AS shuttle-builder
 ARG CARGO_PROFILE
 ARG prepare_args
 COPY builder/prepare.sh /prepare.sh
 RUN /prepare.sh "${prepare_args}"
-COPY --from=chef-builder /build/target/${CARGO_PROFILE}/shuttle-builder /usr/local/bin/service
+COPY --from=chef-builder /build/target/${CARGO_PROFILE}/shuttle-builder /usr/local/bin
+ENTRYPOINT ["/usr/local/bin/shuttle-builder"]
 FROM shuttle-builder AS shuttle-builder-dev
 
-FROM shuttle-crate-base AS shuttle-deployer
+
+#### DEPLOYER
+ARG RUSTUP_TOOLCHAIN
+FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-bookworm AS shuttle-deployer
```

Comment:
Will this be a possible breaking change for users who relied on dependencies installed in the buster image?

Reply:
It looks like they install the same dependencies :+1:

Reply:
Just newer versions. There can be breaking changes though.

Reply:
I think upgrades are okay, as long as we notify users in the release notes.